app/controllers/__init__.py

```diff
@@ -13,8 +13,11 @@
 )
 
 # Import and add namespaces
-from .heatmap_controller import heatmap_ns as heatmap_ns
-from .score_controller import score_ns as score_ns
+ns_heatmap: Namespace = Namespace("heatmap", description="")
+from .heatmap_controller import HeatmapController
 
-api.add_namespace(heatmap_ns)
-api.add_namespace(score_ns)
+ns_score: Namespace = Namespace("score", description="")
```

Comment:
```suggestion
ns_score: Namespace = Namespace("f1-score", description="")
```
Spezifische Benennung, wenn wir zwischen `f1-score` und `Jaccard-Similarity` unterscheiden